dataland-frontend/src/components/forms/parts/elements/basic/UploadDocumentsFormWithComment.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div class="form-field" :data-test="name">
+    <FormKit v-model="baseDataPoint" type="group" name="dataSource">
+      <UploadDocumentsForm
+        @updatedDocumentsSelectedForUpload="handleDocumentUpdatedEvent"
+        ref="uploadDocumentsForm"
+        :name="name"
+        :more-than-one-document-allowed="false"
+        :file-names-for-prefill="fileNamesForPrefill"
+      />
+      <FormKit type="hidden" name="fileName" v-model="documentName" />
+      <FormKit type="hidden" name="fileReference" v-model="documentReference" />
+      <FormKit
+        type="textarea"
+        name="comment"
```

Comment:
Is thise reflected in the backend? As far as i could see we do not have a comment field for documents in the backend. This must be aligned, otherwise i think this will not work here.

Reply:
![image](https://github.com/d-fine/Dataland/assets/96523011/6979f93c-4623-4d3a-b093-569b8639431c)
changed

Reply:
ok